src/test/java/de/tinf/collections/ArrayUtilsTest.java

```diff
@@ -295,4 +296,34 @@ void testContains() {
         result = ArrayUtils.contains(emptyArray, 1);
         assertFalse(result);
     }
+
+    @Test
+    void testFind() {
+        Integer[] array = { 1, 2, 3, 4, 5 };
+
+        // Test finding an element that exists
+        Predicate<Integer> predicate = x -> x == 3;
+        Integer result = ArrayUtils.find(array, predicate);
+        assertTrue(result != null && result == 3);
```

Comment:
_nitpick:_ 
`result == 3` is the stronger condition `result != null `is therefore not necessary